python/cugraph/cugraph/structure/graph_implementation/simpleGraph.py

```diff
@@ -878,8 +878,8 @@ def _make_plc_graph(self, value_col=None, store_transposed=False, renumber=True)
         self._plc_graph = SGGraph(
             resource_handle=ResourceHandle(),
             graph_properties=graph_props,
-            src_array=self.edgelist.edgelist_df[simpleGraphImpl.srcCol],
-            dst_array=self.edgelist.edgelist_df[simpleGraphImpl.dstCol],
+            src_or_offset_array=self.edgelist.edgelist_df["src"],
+            dst_or_index_array=self.edgelist.edgelist_df["dst"],
```

Comment:
Can we do this instead of hard coding constants?
```suggestion
            src_or_offset_array=self.edgelist.edgelist_df[simpleGraphImpl.srcCol],
            dst_or_index_array=self.edgelist.edgelist_df[simpleGraphImpl.dstCol],
```